modules/core/src/lut.cpp

```diff
@@ -377,6 +377,8 @@ void cv::LUT( InputArray _src, InputArray _lut, OutputArray _dst )
     CV_OVX_RUN(!ovx::skipSmallImages<VX_KERNEL_TABLE_LOOKUP>(src.cols, src.rows),
                openvx_LUT(src, dst, lut))
 
+    CALL_HAL(lut8u, cv_hal_lut8u, src.data, src.step, lut.data, dst.data, dst.step, src.rows, src.cols);
```

Comment:
Multi-channel inputs and tables are supported by original OpenCV function. HAL should have channels and lut_channels parameters.